src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandomArticleVideo.java

```diff
@@ -68,4 +72,16 @@ public void adsArticleVideoAdsMobile() {
   public void adsArticleVideoMOATTrackingMobile() {
     adsArticleVideoMOATTrackingDesktop();
   }
+
+  private void verifySlots(AdsFandomObject fandomPage, Boolean skipTopLeaderboard) {
+    fandomPage.triggerOnScrollSlots();
+
+    if (!skipTopLeaderboard) {
```

Comment:
Is it because there is no TLB on a video page?

Reply:
Good suggestion, changed.